ComplementaryData/modelCuration/Metabolomics_newRxnProp.tsv

```diff
@@ -0,0 +1,22 @@
+RxnID	rev	RxnName	EC	rxnID_kegg	Source
+MNXR100390	0	ATP:D-Gluconate 6-phosphotransferase	2.7.1.12	R01737	rhea:19433
+MNXR123213	0	6-phosphogluconate phosphatase	3.1.3.-;3.1.3.2;3.1.3.29		metacyc:RXN0-5185
+MNXR110535	0	seed:rxn04605			seed:rxn04605

```

Comment:
@feiranl for consistency with [past branches](https://github.com/SysBioChalmers/yeast-GEM/pull/149/files#r213701403), when the reaction name does not exist in any database we should use the kegg ID (in this case `R06790`), can you please change that? `seed:rxn04605` can stay in `rxnNotes` if needed. Also, you can add the KEGG id to the `model.metKEGGID` field

https://www.metanetx.org/equa_info/MNXR110535